MoviesApp/MoviesApp/SecondScreen/FilmInfoPresenter/FilmInfoPresenter.swift

```diff
@@ -42,7 +54,9 @@ final class FilmInfoPresenter: FilmsInfoTableViewPresentable {
             guard let self = self else { return }
             switch result {
             case let .success(actor):
-                self.actors = actor
+                self.realmService.saveActors(actor, id: self.movie.id)
```

Comment:
```suggestion
                self.realmService.saveActors(actor, id: self.movie.id)
```
сохранение в реалв в метод вынеси